### ISSUE TITLE
Configuration options in .example.gemnasium.yml

### ISSUE THREAD

**Issue by ivdma** [gemnasium/toolbelt#32]
It would be nice to have a complete overview of configuration options available for .gemnasium.yml as they differ from environment variables.

**Comment by gravis** [comment]
Good idea, we will update the example file to match the env vars. Thanks for the suggestion!
